tests/wicked/before_test.pm

```diff
@@ -16,10 +16,12 @@ use warnings;
 use testapi;
 use utils qw(zypper_call systemctl file_content_replace);
 use network_utils qw(iface setup_static_network);
+use serial_terminal qw(add_serial_console);
 
 sub run {
     my ($self, $ctx) = @_;
     $self->select_serial_terminal;
+    add_serial_console('hvc1');
```

Comment:
We could call it somewhere generic with a condition of  `BACKEND==qemu` and `VIRTIO_CONSOLE_NUM>1` and `is SLES`. But I didn't found a good place.